app/src/ui/welcome/configure-git.tsx

```diff
@@ -7,6 +7,7 @@ import { Button } from '../lib/button'
 interface IConfigureGitProps {
   readonly accounts: ReadonlyArray<Account>
   readonly advance: (step: WelcomeStep) => void
+  readonly done: () => void
```

Comment:
I realize this was the already an establish verbage.. but `done` feels like a weird word for an action and took me a second to read it. Feel free to ignore this (probably just me).

Reply:
Yep, I agree 😆  But as you said I basically copied/moved this from the usage stats screen 😅  So I'm not gonna change it now if that's ok!

Reply:
As you know, I going to tell you the right thing, I am not going to wrong Turn!